.github/ISSUE_TEMPLATE/tracking.md

```diff
@@ -0,0 +1,50 @@
+---
+name: "📊 Tracking Issue"
+title: "📊Tracking: [service]"
+about: Add a module tracking issue (for project management)
+labels: management/tracking
+---
+
+### Add your +1 👍  to help us prioritize high-level constructs for this service
+--- 
+
+### Overview:
+<!-- 
+Summary of the service (leverage the service’s product page for the text) and a link to the relevant AWS Docs. This should be the same text that we put at the top of the package’s README.db. Also include a link to the service’s CDK Construct Library API reference page. 
+-->
+
+
+
+
+[AWS Docs](url)  
+[CDK API Reference](url)
+
+### Implementation:
+<!-- 
+Checklist of use cases, constructs, features (such as grant methods) that will ship in this package 
+(not required until the issue is added to the public roadmap)
+-->
+
+
+
+
+### Related Issues:
+<!-- 
+e.g. checklist of links to feature requests, bugs, and PRs that are in scope for GA release of this module 
+(not required until the issues is added to the public roadmap)
+-->
+
+
+
+
+<!-- 
+Labels to add:
+    - management/tracking
+    - service/[name] (create new labels if they don’t already exist)
+    - status/needs-design (if cfn-only)
+    - management/roadmap (when added to the roadmap)
+    - status/in-progress (when added to “working on it” column of the roadmap)
```

Comment:
I don't mind this being here but people are not going to see it once an issue has been opened. This information needs to go elsewhere.

Also, why do we need these labels? If the roadmap is a kanban board, this should be shown in the 'Projects' section of this issue.

Reply:
It's a reminder for whoever creates the issue to add the appropriate labels.

@ccfife why did we want these labels?

Reply:
@nija-at We want GitHub labels added to the tracking issue after it is created, so that we can filter the issue list to specific modules. For example, we'd add the service/Kinesis label to the tracking issue for the Kinesis service, so we can easily filter the tracking issue and all Kinesis-related bugs, feature requests, and PRs in one list.

The other 3 are less important, but all indicate the current status of the module in general.

I agree that the tracking issues' position on the roadmap also indicates status, but my assumption is that not everyone will use the roadmap. Some people will just want to search/filter the issue list directly and these labels will help with that.